next.json

```diff
@@ -414,9 +438,84 @@
             }
         ]
     },
+    "depth stencil state": {
+        "category": "object"
+    },
+    "depth stencil state builder": {
+        "category": "object",
+        "methods": [
+            {
+                "name": "get result",
+                "returns": "depth stencil state"
+            },
+            {
+                "name": "set depth enabled",
```

Comment:
Both for stencil and depth there are two separate aspects: the test and the writing to the framebuffer. For the depth, ComparisonMode::Always is equivalent to disabling the depth test but not the depth write, which is controlled separately.

We should do the same for stencil: ComparisonMode::Always is equivalent to disabling the stencil test and having all stencil ops be Keep is equivalent to disabling the stencil write. These should be the default.